ffi/tests/ctests/src/tests.c

```diff
@@ -0,0 +1,221 @@
+#ifdef NDEBUG
+#undef NDEBUG
+#endif
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <assert.h>
+
+#include <wirefilter.h>
+
+#define WIREFILTER_STRING(v, s) {(v).data = (const unsigned char *)(s); (v).length = strlen(s);}
+
+wirefilter_externally_allocated_str_t wirefilter_init_string(const char *s) {
```

Comment:
This should probably be just `wirefilter_string` (it doesn't init, just returns it). Also, it should probably be `static` as it's a local helper that is not meant to be exported.

Reply:
Done!